grammar/Graql.g4

```diff
@@ -33,13 +36,13 @@ query                 :   query_define    |   query_undefine
                       |   query_match_group |   query_match_group_agg
                       |   query_compute   ;
 
-query_define          :   DEFINE      variable_types ;
-query_undefine        :   UNDEFINE    variable_types ;
+query_define          :   DEFINE      definable+  ;
+query_undefine        :   UNDEFINE    definable+  ;
 
 query_insert          :   MATCH       patterns      INSERT  variable_things
                       |                             INSERT  variable_things     ;
 query_delete          :   MATCH       patterns      DELETE  variable_things     ;
-query_match           :   MATCH       patterns            ( filters )?          ;
+query_match           :   MATCH       patterns            ( filters )           ;
```

Comment:
note: this was removed because having a `?` here as well as on each optional filter caused Antlr warnings

Reply:
Did it not break the parser?

Reply:
why would it? each sub-filter is optional so this `filters` is optional as well

Reply:
okay